src/org/usfirst/frc/team166/robot/subsystems/Intake.java

```diff
@@ -1,18 +1,35 @@
 package org.usfirst.frc.team166.robot.subsystems;
 
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.Victor;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 /**
  *
  */
 public class Intake extends Subsystem {
-    
-    // Put methods for controlling this subsystem
-    // here. Call these from Commands.
-
-    public void initDefaultCommand() {
-        // Set the default command for a subsystem here.
-        //setDefaultCommand(new MySpecialCommand());
-    }
-}
 
+	DoubleSolenoid Actuator;
+	Victor IntakeSIM;
+
+	public Intake() {
+
+	}
+
+	public void ToggleIntakeMotor() {
```

Comment:
The public methods (all methods, really) should have names that describe WHAT they do, not HOW they do it. So, `startCollecting()` for instance, instead of `toggleIntakeMotor()`.

(On that note, the convention in Java is to do `lowerCamelCase` instead of `UpperCamelCase` for methods)